File: Iniciante/URI 1008.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val funcionarios = leitor.nextInt()
        val horas = leitor.nextInt()
        val valorPorHora = leitor.nextDouble()
        val salario = valorPorHora * horas
        println("NUMBER = " + funcionarios)
        printf("SALARY = U$ %.2f\n", salario)
    }

}

File: Iniciante/URI 1008.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val funcionarios: Int = leitor.nextInt();
    val horas: Int = leitor.nextInt();
	val valorPorHora: Double = leitor.nextDouble();
	val salario: Double = horas * valorPorHora;
    
    println("NUMBER = " + funcionarios);
    println(String.format("SALARY = U$ %.2f", salario));
}